src/content/developers/tutorials/monitoring-geth-with-influxdb-and-grafana/index.md

```diff
@@ -0,0 +1,136 @@
+Basic setup specifies endpoint where InfluxDB is reachable and authentication for the database.
+
+```
+geth --metrics --metrics.influxdb --metrics.influxdb.endpoint "http://0.0.0.0:8086" --metrics.influxdb.username "geth" --metrics.influxdb.password "chosenpassword"
+```
+This flags can be appended to a command starting the client or saved to the configuration file.
+
+You can verify that Geth is successfully pushing data, for instance by listing metrics in database. In InfluxDB shell:
+```
+use geth
+show measurements
+```
+## Setting up Grafana {#setting-up-grafana}
+
+Next step is installing Grafana which will interpret data graphically. Follow installation process for your environment in Grafana documentation. Make sure to install OSS version if you don't want otherwise. 
+Example installation steps for Debian distributions using repository: 
+```
+curl -tlsv1.3 --proto =https -sL https://packages.grafana.com/gpg.key | sudo apt-key add - 
+echo "deb https://packages.grafana.com/oss/deb stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+sudo apt update
+sudo apt install grafana
+sudo systemctl enable grafana-server
+sudo systemctl start grafana-server
+```
+Successfully installed Grafana is running, by default is reachable at `localhost:3000`.
+Use preferred browser to access it and login with default credentials user ‘admin’ and password ‘admin’. You will be prompted to change the default password, type your new password and save. 
+
+![](./grafana1.png)
+
+You will be redirected to default Grafana home page. First thing to setup in Grafana is source of data. Click on on configuration icon on left bar and select Data sources. 
+
+![](./grafana2.png)
+
+There aren't any data sources created yet, click on Add data source to define one. 
+
+![](./grafana3.png)
+
+For this setup, select InfluxDB option and proceed. 
+
+![](./grafana4.png)
+
+Data source configuration is pretty straight forward if you are running tools on the same machine. You need to set address of InfluxDB and details for accessing database. Refer to the picture below.  
+
+![](./grafana5.png)
+
+If everything is filled correctly and InfluxDB is reachable, click on Save & Test and confirmation that source is working will pop up. 
+
+![](./grafana6.png)
+
+Grafana is set up to read data from InfluxDB. Now you need to create dashboard which will interpret and display it. Dashboards properties are encoded in JSON files which can be created by anybody and easily imported. On left bar, click on Create and Import. 
+
+![](./grafana7.png)
+
+!!Add dashboard to grafana website for easier import? 
+
+Dashboard for monitoring Geth client is provided, just copy [this JSON file](./gethdashboard.json) and insert it to Import page in Grafana. After successfully saving the dashboard, it should look like this. 
```

Comment:
For a Geth monitoring dashboard, just copy [this JSON file](./gethdashboard.json) and insert it in the "Import page" in Grafana. After saving the dashboard, it should look like this:

Reply:
Just a note, this link didn't work correctly for me in the netlify preview, getting parsed to `/en/{hash}/gethdashboard.json`... Can take a look at this when the rest is finalized.